lib/node/file.ts

```diff
@@ -15,7 +15,7 @@ export class File extends Node {
 	 * Returns a clone of the file
 	 */
 	clone(): File {
-		return new File(this.data)
+		return new File(this._data, this._knownDavService)
```

Comment:
But this is no longer a clone it has the same internal data object. `structuredClone` will remove any dependencies and makes the object transferable.